### ISSUE TITLE
Skip ng-form

### ISSUE THREAD

**Issue by ShMcK** [formly-js/angular-formly#167]
```html
<div container="row">
    <ng-form>
        <div class="col4">Row Element</div>
    </ng-form>
</div>
```
A way for a field to opt out of `<ng-form>` in order to enable rows/columns.

**Comment by kentcdodds** [comment]
Formly doesn’t wrap every `formly-field` in an `ng-form`, but it does wrap every `formly-form` in an `ng-form` is that what you’re talking about? If you'd like to skip the `ng-form`, then I think this is doable. Especially because we're using es6 template strings it should be a breeze. I'll just have to move the template from a file to a function and allow you to specify an attribute that says that you don't want it in an `ng-form`. Should be very easy.

**Comment by kentcdodds** [comment]
The api would probably be something like this:

```html
<formly-form model="vm.model" fields="vm.fields" form="vm.form" no-ng-form></formly-form>
```

Though, thinking about it now, I'm not sure why the presence of `ng-form` prevents rows/columns. Can't you just style the `ng-form`? You can add your own classes to the `formly-form` and it will be applied to the `ng-form`.

**Comment by kentcdodds** [comment]
Closed with 3.0.3 208b851ebe886dd61c9311db08244bb330a91ce3
